### ISSUE TITLE
Could not process database: pq: column "queryid" does not exist

### ISSUE THREAD

**Issue by ghidinelli** [pganalyze/collector#13]
Trying to install pganalyze on Postgres 9.4.10 on a new CentOS 6 install and getting this when testing:

```
[root contrib]# pganalyze-collector --test -v
2017/01/05 14:23:27 V Did not open state file: open /var/lib/pganalyze-collector/state: no such file or directory
2017/01/05 14:23:28 V [default] Detected PostgreSQL Version 90410 (PostgreSQL 9.4.10 on x86_64-unknown-linux-gnu, compiled by gcc (GCC) 4.4.7 20120313 (Red Hat 4.4.7-17), 64-bit)
2017/01/05 14:23:28 E [default] Error collecting pg_stat_statements
2017/01/05 14:23:28 E [default] Could not process database: pq: column "queryid" does not exist
```

I have the required settings in postgresql.conf (and am confident they are right because I've been using the same postgresql.conf file successfully with pganalyze on another server).

**Comment by lfittl** [comment]
@ghidinelli This typically happens when you've upgraded Postgres without updating the views for pg_stat_statements.

Could you try running `ALTER EXTENSION pg_stat_statements UPDATE` and then retry the collector test run?

**Comment by ghidinelli** [comment]
Thanks @lfittl - that did indeed fix it. I wasn't aware that was a thing. I ran it with `-v` and did still see some disk usage collection problems though:

```
2017/01/06 09:44:45 V Did not open state file: open /var/lib/pganalyze-collector/state: no such file or directory
2017/01/06 09:44:45 V [default] Detected PostgreSQL Version 90410 (PostgreSQL 9.4.10 on x86_64-unknown-linux-gnu, compiled by gcc (GCC) 4.4.7 20120313 (Red Hat 4.4.7-17), 64-bit)
2017/01/06 09:44:46 V [default] Selfhosted/System: Could not run helper process: fork/exec /usr/bin/pganalyze-collector-helper: exec format error
2017/01/06 09:44:46 V [default] Selfhosted/System: Failed to get disk partition usage stats: input/output error
2017/01/06 09:44:46 V [default] Selfhosted/System: Failed to get disk partition usage stats: input/output error
2017/01/06 09:44:46 V [default] Successfully prepared S3 request - size of request body: 0.1134 MB
2017/01/06 09:44:46 I [default] Test submission successful (588 KB received) - proceed running the collector as a daemon
```

**Comment by lfittl** [comment]
@ghidinelli That is interesting, its odd you are getting both the `exec format error` as well as `input/output error`.

Two things:

* Whats the system you are on? (Operating System, 32-bit vs 64-bit, virtual machine or not)
* What happens if you run `/usr/bin/pganalyze-collector-helper status`

Thanks!

**Comment by ghidinelli** [comment]
Well, oddly, this happens: 

```
# /usr/bin/pganalyze-collector-helper status
bash: /usr/bin/pganalyze-collector-helper: cannot execute binary file
```

This is my env:

```
[root@ewhserver2308 pukkasoft]# uname -a
Linux ewhserver2308.edgewebhosting.net 2.6.32-642.11.1.el6.x86_64 #1 SMP Fri Nov 18 19:25:05 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
[root@ewhserver2308 pukkasoft]# cat /etc/issue.net 
CentOS release 6.8 (Final)
```

**Comment by lfittl** [comment]
@ghidinelli FYI, this should be resolved with collector release 0.9.10.
